src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { Link, Outlet, useLocation } from 'react-router-dom';
+import MUITabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Pathnames from 'constants/pathnames';
+import Tabs from 'constants/tabs';
+
+function Layout() {
+  const location = useLocation();
+
+  const a11yProps = (index: number) => ({
+    id: `tab-${index}`,
+    'aria-controls': `tabpanel-${index}`,
+  });
+
+  return (
+    <>
+      <MUITabs
+        value={
+          location.pathname === (Pathnames.HISTORICAL as string)
```

Comment:
Make this check stricter: check all variants and if no match found throw an error
Also extract this check of value to function inside layouy